src/components/elements/Footer.jsx

```diff
@@ -33,10 +33,11 @@ function Footer() {
           <ul className="grid lg:grid-rows-3 lg:gap-4">
             <li className="mb-2"><a href="https://www.dataumbrella.org">Data Umbrella</a></li>
             <li className="mb-2"><a href="/privacy">Privacy</a></li>
-            <li className="mb-2">Terms & Conditions</li>
+            <li className="mb-2"><a href="/terms">Terms & Conditions</a></li>
```

Comment:
Comment (blocking):
Go ahead and change all of these links right here specifically, to use the react router Link element
Only use anchor links for external pages when writing react (unless its nextjs)